data/data-pipeline/data_pipeline/etl/sources/michiganejscreen/etl.py

```diff
@@ -0,0 +1,67 @@
+import pandas as pd
+
+from data_pipeline.etl.base import ExtractTransformLoad
+from data_pipeline.utils import get_module_logger
+from data_pipeline.config import settings
+
+logger = get_module_logger(__name__)
+
+
+class MichiganEnviroScreenETL(ExtractTransformLoad):
+    """
+    Michigan EJ Screen class that ingests dataset represented
+    here: https://www.arcgis.com/apps/webappviewer/index.html?id=dc4f0647dda34959963488d3f519fd24
+    """
+
+    def __init__(self):
+        self.MIEJSCREEN_S3_URL = (
+            settings.AWS_JUSTICE40_DATASOURCES_URL
+            + "/michigan_ejscore_12212021.csv"
+        )
+
+        self.CSV_PATH = self.DATA_PATH / "dataset" / "michiganejscreen"
+
+        # Definining some variable names
+        self.MIEJSCREEN_SCORE_FIELD = "michiganejscreen_score"
+        self.MIEJSCREEN_PERCENTILE_FIELD = "michiganejscreen_percentile"
+        self.MIEJSCREEN_PRIORITY_COMMUNITY_FIELD = (
+            "michiganejscreen_priority_community"
+        )
+
+        self.MIEJSCREEN_PRIORITY_COMMUNITY_THRESHOLD = 0.75
```

Comment:
Where does this cutoff come from? Is it suggested by the Michigan literature / authors?